README 2.md

```diff
@@ -0,0 +1,39 @@
+# Project Name
```

Comment:
Fill this section in eventually. I don't think it's needed right now.

Reply:
You are right. I think it created this file when I copied the repository, but I am not sure how.